electron/main/index.ts

```diff
@@ -191,6 +191,22 @@ async function createWindow() {
     return await baseCoin.recover(parameters);
   });
 
+  ipcMain.handle(
+    'wrongChainRecover',
+    async (event, sourceCoin, destinationCoin, parameters) => {
+      try {
+        const sourceBaseCoin = sdk.coin(sourceCoin) as AbstractUtxoCoin;
+        const destinationCoinBase = sdk.coin(
+          destinationCoin
+        ) as AbstractUtxoCoin;
```

Comment:
no way to re-write this without `as` ? Could you try to `type narrow` this?

Reply:
Fixed